code/web/src/modules/user/Profile.js

```diff
@@ -9,54 +9,233 @@ import { Link } from 'react-router-dom'
 import { Grid, GridCell } from '../../ui/grid'
 import { H3, H4 } from '../../ui/typography'
 import Button from '../../ui/button'
-import { grey, grey2 } from '../../ui/common/colors'
+import ImageTile from '../../ui/image/Tile'
+import Card from '../../ui/card/Card'
+import { level3 } from '../../ui/common/shadows'
+import { white, black, grey, grey2 } from '../../ui/common/colors'
 
 // App Imports
+import { APP_URL } from '../../setup/config/env'
 import userRoutes from '../../setup/routes/user'
 import { logout } from './api/actions'
+import { getListByUser } from '../subscription/api/actions'
+import { routeImage } from '../../setup/routes'
+
+
+const userImage = `https://media.newyorker.com/photos/5e49bf473399bf0008132231/1:1/w_2539,h_2539,c_limit/Kenseth-CatProfile.jpg`
+
+const mockProductHistory = [
+  {
+    id:8,
+    name:"T-Shirt for Men - Grey",
+    slug:"t-shirt-for-men-grey",
+    description:"A very nice grey t-shirt for men.",
+    image:"/images/stock/t-shirt-male-2.jpg",
+    createdAt:"1616449062163",
+    updatedAt:"1616449062163",
+    kept: true
+  },
+  {
+    id:4,
+    name:"Watch for Men",
+    slug:"watch-for-men",
+    description:"A very nice watch for men.",
+    image:"/images/stock/watch-male.jpg",
+    createdAt:"1616449062163",
+    updatedAt:"1616449062163",
+    kept: false
+  }
+]
 
 // Component
-const Profile = (props) => (
-  <div>
-    {/* SEO */}
-    <Helmet>
-      <title>My Profile - Crate</title>
-    </Helmet>
-
-    {/* Top title bar */}
-    <Grid style={{ backgroundColor: grey }}>
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
-        <H3 font="secondary">My profile</H3>
-      </GridCell>
-    </Grid>
-
-    <Grid>
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
-        <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
-
-        <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.email}</p>
-
-        <Link to={userRoutes.subscriptions.path}>
-          <Button theme="primary">Subscriptions</Button>
-        </Link>
-
-        <Button theme="secondary" onClick={props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
-      </GridCell>
-    </Grid>
-  </div>
-)
+class Profile extends PureComponent {
+
```

Comment:
Updated this from a functional component to a class component, because it's going to hold a form.